packages/core/src/api/exporters/html/__snapshots__/codeBlock/defaultLanguage/internal.html

```diff
@@ -1 +1 @@
-<div class="bn-block-group" data-node-type="blockGroup"><div class="bn-block-outer" data-node-type="blockOuter" data-id="1"><div class="bn-block" data-node-type="blockContainer" data-id="1"><div class="bn-block-content" data-content-type="codeBlock"><pre><code class="bn-inline-content language-javascript">console.log('Hello, world!');</code></pre></div></div></div></div>
\ No newline at end of file
+<div class="bn-block-group" data-node-type="blockGroup"><div class="bn-block-outer" data-node-type="blockOuter" data-id="1"><div class="bn-block" data-node-type="blockContainer" data-id="1"><div class="bn-block-content" data-content-type="codeBlock"><pre><code class="bn-inline-content">console.log('Hello, world!');</code></pre></div></div></div></div>
```

Comment:
is it easy / needed to add tests whether the exporting / importing to / from md / html still works for custom languages?

Reply:
I think we have some existing tests for this: https://github.com/TypeCellOS/BlockNote/blob/013818d40a2fedaf192ddc7021a8457b2eac37bc/packages/core/src/api/exporters/html/__snapshots__/codeBlock/python/internal.html#L0-L1

Just didn't need to modify it because it was only the default language that changed (it went from being `javascript` by default to `text`)